### PR TITLE
WFLY-16938 Drop source transformation for ejb3 subsystem

### PR DESCRIPTION
**Pull request by chengfang**
https://issues.redhat.com/browse/WFLY-16938

### CODE REVIEW

File: ejb3/pom.xml [hunk 1]
```diff
@@ -307,11 +384,25 @@ projects that depend on this project.-->
...
+        <dependency>
+            <groupId>org.infinispan</groupId>
+            <artifactId>infinispan-commons-jakarta</artifactId>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.infinispan</groupId>
+            <artifactId>infinispan-core-jakarta</artifactId>
+            <scope>test</scope>
+        </dependency>
```
Comment (pferraro) [inline ejb3/pom.xml:405]: @chengfang I don't think these test dependencies are needed any longer.  Can you verify?

Comment (chengfang) [reply ejb3/pom.xml:405]: @pferraro You're right. I'll let the current test run to complete and remove the 2 deps.

**Review by tadamski — APPROVED** [review]
LGTM

**Review by bstansberry — APPROVED** [review]

**Comment by bstansberry** [comment]
Thanks @chengfang

**Pull request merged**